packages/codepush/README.md

```diff
@@ -0,0 +1,50 @@
+# React-Native Monitoring for AppCenter CodePush
+
+## Setup
+
+**Note**: This package is an integration for the [`react-native-code-push`][1] library. Before using it, you must first install and setup the core [`mobile-react-native`][2] SDK.
+
+To install with NPM, run:
+
+```sh
+npm install @datadog/mobile-react-native-code-push
+```
+
+To install with Yarn, run:
+
+```sh
+yarn add @datadog/mobile-react-native-code-push
+```
+
+## Initialize the SDK
+
+To initialize the SDK, use `DatadogCodepush.initialize` instead of `DdSdkReactNative.initialize`:
+
+```js
+import { DdSdkReactNativeConfiguration } from '@datadog/mobile-react-native';
+import { DatadogCodepush } from '@datadog/mobile-react-native-code-push';
+
+const config = new DdSdkReactNativeConfiguration(
+    '<CLIENT_TOKEN>',
+    '<ENVIRONMENT_NAME>',
+    '<RUM_APPLICATION_ID>',
+    true, // track User interactions (e.g.: Tap on buttons. You can use 'accessibilityLabel' element property to give tap action the name, otherwise element type will be reported)
+    true, // track XHR Resources
+    true // track Errors
```

Comment:
```suggestion
    true, // track user interactions (such as a tap on buttons). You can use the 'accessibilityLabel' element property to give the tap action a name, otherwise the element type is reported
    true, // track XHR resources
    true // track errors
```

Reply:
same comment as above